mod/deckorate_graphql/README.md

```diff
@@ -1,4 +1,55 @@
 <!--
 # @title README - mod: graphql
```

Comment:
can you add deckorate to the title so that in generated docs it shows up with a different name from the other graphql mod?

(the reason we have these sections is for yard sites like docs.decko.org. We don't have a wikirate one yet but will soon I hope)